bayesian_cnn_prometheus/preprocessing/data_generator.py

```diff
@@ -46,21 +71,19 @@ def get_test(self):
     def get_valid(self):
         return self._get_data_generator(DatasetType.VALID, self.batch_size)
 
-    def _get_data_generator(self, dataset_type: DatasetType, batch_size: int):
+    def _get_data_generator(self, dataset_type: str, batch_size: int):
```

Comment:
Dlaczego str a nie DatasetType?

Reply:
`DatasetType` to klasa, a nie `enum`, przez co `mypy` się czepiał do tego, że przekazujemy `str`, a nie instancję klasy